CHANGELOG.md

```diff
@@ -1,54 +1,61 @@
 # CHANGELOG
 
-## 0.14.0 (unreleased)
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-**all**
+## [Unreleased]
 
-- The `query` enpoint is now optional. It is still hightly recommended to expose
-  it an almost any use case though.
+### Added
 
-**cosmwasm-std**
+- cosmwasm-vm: Add PinnedMemoryCache. ([#696])
 
-- Remove `from_address` from `BankMsg::Send`, as it always sends from the
-  contract address, and this is consistent with other `CosmosMsg` variants.
+### Changed
 
-**cosmwasm-vm**
+- all: The `query` enpoint is now optional. It is still hightly recommended to
+  expose it an almost any use case though.
+- cosmwasm-std: Remove `from_address` from `BankMsg::Send`, as it always sends
+  from the contract address, and this is consistent with other `CosmosMsg`
+  variants.
+- cosmwasm-vm: Avoid serialization of Modules in `InMemoryCache`, for
+  performance. Also, remove `memory_limit` from `InstanceOptions`, and define it
+  instead at `Cache` level (same memory limit for all cached instances).
+  ([#697])
```

Comment:
So, we add these then? I think they are useful, particularly for terse Changelog entries.

Reply:
I don't have a strong opinion on those. I left what was there before. I think issue numbers are easier to use than PR numbers because you know the number early on. I'm open to discuss this.

Reply:
I'm in favour of adding them. But, this could be optional.

My take would be: If you think the Changelog entry is enough for documentation, don't add an issue / pr number. But if you think more info could be warranted, and don't want to put all that in the Changelog, add it. Particularly if the discussions / info in the issue / pr are relevant / useful.

Regarding issue number vs. pr number, given that they are treated indistinctly in github, I would use any, indistinctly. That also allows you to add an (already existing) issue number, when you don't have the pr number yet. Plus, they are usually linked, so, it's the same to add any of them.

Reply:
Sounds good